lib/engine/game/g_1893.rb

```diff
@@ -0,0 +1,185 @@
+# frozen_string_literal: true
+
+require_relative '../config/game/g_1893'
+require_relative 'base'
+
+module Engine
+  module Game
+    class G1893 < Base
+      register_colors(
+        gray70: '#B3B3B3',
+        gray50: '#7F7F7F'
+      )
+
+      load_from_json(Config::Game::G1893::JSON)
+      AXES = { x: :number, y: :letter }.freeze
+
+      GAME_LOCATION = 'Cologne, Germany'
+      GAME_RULES_URL = 'https://boardgamegeek.com/filepage/188242/1824-english-rules'
+      GAME_DESIGNER = 'Leonhard Orgler & Helmut Ohley'
```

Comment:
this is all wrong

Reply:
@tobymao I did some copy pasting from 1824 when I created this so some things I have not updated yet. Will try to fix this as I go along.